cmd/chaos-builder/impl.go

```diff
@@ -35,12 +35,18 @@ func (in *{{.Type}}) IsDeleted() bool {
 	return !in.DeletionTimestamp.IsZero()
 }
 
-// IsPaused returns whether this resource has been paused
-func (in *{{.Type}}) IsPaused() bool {
-	if in.Annotations == nil || in.Annotations[PauseAnnotationKey] != "true" {
-		return false
+// GetPause returns the annotation when if chaos is paused
+// return empty when annotations is not set, or pause key is set to empty
+func (in *{{.Type}}) GetPause() string {
+	if in.Annotations == nil {
+		return ""
 	}
-	return true
+	return in.Annotations[PauseAnnotationKey]
+}
+
+// SetPause set the pausetime of annotation. Empty meams not paused
+func (in *{{.Type}}) SetPause(s string) {
```

Comment:
Why not add a single function to recover pause action?

Reply:
Good point! Fixed thanks!